SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -595,6 +596,13 @@ Buffer {
 		^[\b_query, bufnum]
 	}
 
+	sampleRate_ { | newValue |
```

Comment:
All the other methods use the `arg` style.

Reply:
@Spacechild1 I don't know how reviews work here - do I close all corrected 'conversations'?

Reply:
There is a button "Resolve conversation". But I'm not entirely sure myself who is responsible for clicking it :) I just went ahead and resolved the conversation.

Reply:
ok I'll resolve the non-ambiguous ones and will wait for your re-review in silence :)